docs/CaliptraIntegrationSpecification.md

```diff
@@ -599,18 +599,26 @@ The following table describes SoC integration requirements.
 | Deobfuscation Key                | Rotation of the deobfuscation key (if not driven through PUF) between silicon steppings of a given product (for example, A0 vs. B0 vs. PRQ stepping) is dependent on company-specific policies.                                           | Statement of conformance | Required by UDS and Field Entropy threat model    |
 | Deobfuscation Key                | SoC backend flows should not insert deobfuscation key flops into the scan chain.                                                                                                                                                          | Synthesis report         | Required by UDS and Field Entropy threat model    |
 | Deobfuscation Key                | For defense in depth, it is strongly recommended that debofuscation key flops are not on the scan chain. <br> Remove the following signals from the scan chain: <br> cptra_scan_mode_Latched_d <br> cptra_scan_mode_Latched_f <br> field_storage.internal_obf_key | Statement of conformance | Caliptra HW threat model |
+| Obfuscation Key                  | SOC shall ensure that obfuscation key is available (and wires are stable) before Caliptra reset is de-asserted. | Statement of conformance | Functionality and Security |
```

Comment:
I believe we settled on to "obfuscation key" as the chosen name, right ?

Reply:
Most of the doc referenced deobfuscation, a few places use obfuscation. I only noticed since this change is right next to a bunch of deobfuscation uses. I don't have a preference either way, just thought we should update the usage for consistency everywhere in the doc.

Reply:
I asked @bharatpillilli a few months ago and we settled on "obfuscation key"

Reply:
Perfect. Either this PR or in another one we should find and replace the references to deobfuscation key possibly.

Reply:
I believe Deobfuscation is the correct term. The fuse value is considered to be an obfuscated UDS Seed, so it needs to be deobfuscated to be useable.

Reply:
The main spec refers to this only as "obfuscation key." I agree, this document should be cleaned to make the terminology consistent

Reply:
Pushed a change to replace all references to "deobfuscation key" with "obfuscation key".